caasp-cert-manager-acmesolver-image/caasp-cert-manager-acmesolver-image.kiwi

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- OBS-AddTag: caasp/v5/cert-manager-acmesolver:%%PKG_VERSION%%-rev<VERSION> caasp/v5/cert-manager-acmesolver:%%PKG_VERSION%%-rev<VERSION>-build<RELEASE> -->
+
+<image schemaversion="6.9" name="caasp-cert-manager-acmesolver-image" xmlns:suse_label_helper="com.suse.label_helper">
+  <description type="system">
+    <author>SUSE Containers Team</author>
+    <contact>containers@suse.com</contact>
+    <specification>cert-manager-acmesolver running on a SLES15 SP2 container guest</specification>
+  </description>
+  <preferences>
+    <type
+      image="docker"
+      derived_from="obsrepositories:/suse/sle15#15.2">
+      <containerconfig
+        name="caasp/v5/cert-manager-acmesolver"
+        tag="%%PKG_VERSION%%"
+        maintainer="SUSE Containers Team &lt;containers@suse.com&gt;"
+        user="nobody">
```

Comment:
for 15-SP2, do we need to add nobody package when use user nobody?

Reply:
nice catch!